sigs.yaml

```diff
@@ -761,6 +761,9 @@ sigs:
     - name: cluster-api-provider-openstack
       owners:
       - https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-openstack/master/OWNERS
+    - name: cluster-api-provider-digitalocean
```

Comment:
nit: I think this should go just below aws (and above gcp) to maintain alphabetical order.

Reply:
Fixed :)